pkg/cmd/mom/output_resources_command.go

```diff
@@ -0,0 +1,68 @@
+package mom
+
+import (
+	"context"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryoutputresources"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewOutputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryoutputresources.NewOutputResourcesCommand(RunOutputResources, streams)
+}
+
+func RunOutputResources(ctx context.Context) (*libraryoutputresources.OutputResources, error) {
+	return &libraryoutputresources.OutputResources{
+		ConfigurationResources: libraryoutputresources.ResourceList{
+			ExactResources: []libraryoutputresources.ExactResource{
+				exactResource("config.openshift.io", "ingresses", "", "cluster"),
+			},
+		},
+		ManagementResources: libraryoutputresources.ResourceList{
+			ExactResources: []libraryoutputresources.ExactResource{
+				exactResource("config.openshift.io", "clusteroperators", "", "authentication"),
+				exactResource("openshift.openshift.io", "authentications", "", "cluster"),
+			},
+		},
+		UserWorkloadResources: libraryoutputresources.ResourceList{
+			ExactResources: []libraryoutputresources.ExactResource{
+				exactResource("", "secrets", "openshift-authentication", "v4-0-config-system-session"),
+				exactResource("", "secrets", "openshift-authentication", "v4-0-config-system-ocp-branding-template"),
+				exactResource("", "serviceaccounts", "openshift-authentication", "oauth-openshift"),
+				exactResource("apps", "deployments", "openshift-authentication", "oauth-openshift"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-browser-client"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-challenging-client"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-cli-client"),
+				exactResource("rbac.authorization.k8s.io", "clusterrolebindings", "", "system:openshift:openshift-authentication"),
+				exactResource("rbac.authorization.k8s.io", "rolebindings", "openshift-config-managed", "system:openshift:oauth-servercert-trust"),
+				exactResource("rbac.authorization.k8s.io", "roles", "openshift-config-managed", "system:openshift:oauth-servercert-trust"),
+			},
+			GeneratedNameResources: []libraryoutputresources.GeneratedNameResource{
```

Comment:
I don't understand the purpose of `GeneratedNameResources`. I checked the mom repo and it is not used.